src/sentry/models/sourcemaperror.py

```diff
@@ -0,0 +1,29 @@
+"""
+A class for tracking specific source map errors, based on EventError
+"""
+
+
+class SourceMapError:
+    # Generic errors
+    UNKNOWN_ERROR = "unknown_error"
+    NO_RELEASE_ON_EVENT = "no_release_on_event"
+
+    _messages = {
+        UNKNOWN_ERROR: "Unknown error",
+        NO_RELEASE_ON_EVENT: "The event is not tagged with a release",
```

Comment:
Afair we don't call `release` a "tag".It is stored internally as such, but it's a completely separate thing conceptually. I'd rename it to something like "The event is missing a release"